dandiapi/api/models/metadata.py

```diff
@@ -3,13 +3,20 @@
 from typing import TYPE_CHECKING
 from uuid import uuid4
 
+from django.conf import settings
+
+from dandiapi import __version__
```

Comment:
#2597 removed `__version__` in favor of `importlib.metadata.version` as a best practice.
```suggestion
import importlib
```